pkg/cmd/pulumi/host.go

```diff
@@ -0,0 +1,234 @@
+// Copyright 2016-2018, Pulumi Corporation.
```

Comment:
Nit:

```suggestion
// Copyright 2016-2020, Pulumi Corporation.
```

Reply:
BTW, I wouldn't bother updating the copyright for any existing files you touched. But new files at least should have the current year.